reverse-array.js

```diff
@@ -0,0 +1,25 @@
+// https://eloquentjavascript.net/04_data.html#i_6xTmjj4Rf5
+
+function reverseArray(array){
+  let newArray = [];
+  for (let item of array){
+    newArray.unshift(item);
```

Comment:
why `unshift`?

Reply:
Soooo it adds each item from the original array to the front of a new array, so the end result is the original array in reverse.

`unshift()` was one of the methods discussed in the chapter so I figured it was meant to be used. Also I wasn't allowed to use `reverse()`.